packages/gatsby-image/src/index.js

```diff
@@ -168,9 +168,19 @@ class Image extends React.Component {
     this.handleRef = this.handleRef.bind(this)
   }
 
+  componentDidMount() {
+    if (this.state.isVisible && typeof this.props.onStartLoad === `function`) {
+      this.props.onStartLoad()
```

Comment:
just checking here - this won't cause firing this "event" twice if image is visible in initial viewarea?

Reply:
good point, I just pushed a change to fire only if the image was not visible before